org.ekstep.mathtext-1.0/editor/mathtext.js

```diff
@@ -754,13 +758,35 @@ angular.module('org.ekstep.mathtext', [])
       return divElement.textContent || divElement.innerText;
     }
 
+     $timeout( function() {
+      /**
+       * @description 
+       * - Responsible for returning the range of Cursor from selection Start
+       *   to End through MOUSE EVENT.
+       * - Click event has more priority than select event.
+       */
+      setTimeout( function() {
+        ecEditor.jQuery( '#advInput' ).select(function(selectTextEvent) {
+           $scope.selectionEnd= selectTextEvent.target.selectionEnd
+           $scope.selectionStart= selectTextEvent.target.selectionStart
+        });
+        ecEditor.jQuery( '#advInput' ).on('click',function() {
+          $scope.selectionStart = $scope.selectionEnd = undefined;
+        }); 
+      },100);
+     })
+    
     $scope.getCursorPosition = function(e){
       var currentPosition = e.target.selectionEnd;
-      if(e.which == 8){
-        $scope.cursorPosition = currentPosition == 0 ? currentPosition : currentPosition -1;
+      if(!_.isUndefined($scope.selectionStart) && !_.isUndefined($scope.selectionEnd)){
```

Comment:
$scope.selectionEnd remove this variable, not required.

Reply:
Remove the variable.

Reply:
removed validation for selection end.